hudi-common/src/main/java/org/apache/hudi/common/table/TableSchemaResolver.java

```diff
@@ -300,21 +273,6 @@ private Option<MessageType> getTableParquetSchemaFromDataFile() {
     }
   }
 
-  public static MessageType convertAvroSchemaToParquet(Schema schema, Configuration hadoopConf) {
```

Comment:
Do we have full test coverage on the public methods in this class?

Reply:
Want to make sure there is no implicit assumption on schema conversions (e.g., making namespace consistent, case sensitivity, etc.), since this PR removes the back-and-forth conversion between Avro and Parquet schema.

Reply:
That seems like a legitimate concern. We do not have many test

Reply:
Our functional tests cover a few schema evolution cases that execute the logic in `TableSchemaResolver`.  Still, we should do more testing before the release to make sure everything still works.